src/mscorlib/ref/mscorlib.csproj

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <PropertyGroup>
+    <AssemblyVersion>4.0.0.0</AssemblyVersion>
+    <ProjectGuid>{E9E2FEE9-3F81-4B59-A24C-0DBFC47C960C}</ProjectGuid>
+    <OutputType>Library</OutputType>
```

Comment:
No, I was planning to do that on my next iteration but can add it here if necessary.